pkg/catalog/catalog_test.go

```diff
@@ -67,6 +67,9 @@ func TestVersionExist(t *testing.T) {
 				return true, testingutil.FakeImageStreams(tt.args.name, tt.args.namespace, tt.args.tags), nil
 			})
 
+			fakeClientSet.ImageClientset.PrependReactor("list", "imagestreams", func(action ktesting.Action) (bool, runtime.Object, error) {
+				return true, testingutil.FakeImageStreams(tt.args.name, "openshift", tt.args.tags), nil
+			})
```

Comment:
Check for `fakeClientSet.ImageClientset.Action` missing

Reply:
Ack adding